pkg/annotations/annotations.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2018 The Containerd Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package annotations
+
+// ContainerType values
+const (
+	// ContainerTypeSandbox represents a pod sandbox container
+	ContainerTypeSandbox = "sandbox"
+
+	// ContainerTypeContainer represents a container running within a pod
+	ContainerTypeContainer = "container"
+
+	// ContainerType is the container type (sandbox or container) annotation
+	ContainerType = "io.kubernetes.cri.ContainerType"
```

Comment:
@mikebrow changed namespace annotations to `io.kubernetes.cri`

Reply:
Can we use `io.kubernetes.cri.container-type` or `io.kubernetes.cri.containertype`.

The Kubernetes naming convention https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md

Reply:
suggest to correct this.